consent/strategy_default.go

```diff
@@ -442,7 +476,7 @@ func (s *DefaultStrategy) verifyAuthentication(w http.ResponseWriter, r *http.Re
 	cookie.Options.Path = s.c.SessionCookiePath(ctx)
 	cookie.Options.SameSite = s.c.CookieSameSiteMode(ctx)
 	cookie.Options.Secure = s.c.CookieSecure(ctx)
-	if err := cookie.Save(r, w); err != nil {
+	if err = cookie.Save(r, w); err != nil {
```

Comment:
Keep errors in ifs scoped to itself to avoid overshadowing:

```
if err := cookie.Save() ...
```

Reply:
I don't understand the difference here.